CONTRIBUTING.md

```diff
@@ -79,43 +90,37 @@ submit a pull request:
    usage presented as doctests when appropriate.
```

Comment:
It's not part of your changes, but shouldn't it be "closes #issue-number" rather than #PR-NUM?

Reply:
Definitely, missed that one.